client/src/components/Table.js

```diff
@@ -0,0 +1,132 @@
+import React from "react";
+import "../css/Table.css";
+import "../css/themePalette.css";
+import { Table, Tabs, Tab } from "react-bootstrap";
+
+
+import { ReactComponent as Logo } from "../img/Logo_CORONATRACKER_Logo.svg";
+import { ReactComponent as TextLogo } from "../img/Logo_CORONATRACKER_Text_Logo.svg";
+
+// Show the current date, this might should be put in App.js or as a permint layout
+const options = {
+  weekday: "long",
+  year: "numeric",
+  month: "long",
+  day: "numeric"
+};
+const today = new Date(),
+  date = today.toLocaleDateString(undefined, options);
```

Comment:
So as it stands now, the Date is already creeated and displayed in `DiagnosticContainer.js` - I felt it may be better to keep in the wrapper component as we might switch views but still want to display the date, and that way we won't have to replicate getting the date across components. Let me know what you think!

Reply:
I agree that it should be in one place. I was thinking if we will make a Layout component that includes navbar, sing-out button, date components like these, things that will be showing everywhere in the app. I talked to @tesla809 and I am thinking about using material-ui . I will try something and come back to you

Reply:
I need to do some reading about "rebase" and will do what you asked

Reply:
I think using the two in parallel for the time being with an iterative shift over to Material UI is a good long-term strategy! It seems to be much more feature-rich